FCWTAPI/PlottingUtils.cs

```diff
@@ -35,6 +35,50 @@ public class PlottingUtils
             }
             return data;
         }
+        /// <summary>
+        /// Generates a double[1000,1000] containing asymmetrical gaussian data
+        /// The first 400 x values are multiplied by 4 and the first 200 y values are multiplied by 3
+        /// This is used to check that plotting functions preserve data orientation
+        /// </summary>
+        /// <returns></returns>
+        public static double[,] GenerateAssymetricalGaussian()
```

Comment:
Asymmetrical is spelled incorrectly. I also think this method and GenerateGaussian need to be moved to a class within the test project. A class should only do one thing, and the purpose of this class is to plot CWT data.

Reply:
Moved this to a new class